addons/markers/functions/fnc_isUniqueMap.sqf

```diff
@@ -0,0 +1,17 @@
+/*
+ * Author: SzwedzikPL
+ * Checks if item is a unique map
+ *
+ * Arguments:
+ * 0: Item class <STRING>
+ *
+ * Return Value:
+ * is a unique map <BOOL>
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_item"];
+
+((toLower _item) find "ace_map_") == 0
```

Comment:
Not exactly, you don't only need to know if the item is a map, but if it's unique